src/models/sentry-scope-proxy.ts

```diff
@@ -18,9 +24,9 @@ export class SentryScopeProxy {
 		this.extra = {};
 	}
 
-	static processEvent(this: void, event: any, hint: any) {
-		if (hint.originalException.sentryScope) {
-			hint.originalException.sentryScope.addTo(event);
+	static processEvent(event: Event, hint?: EventHint | undefined): SentryScopeProxyEvent {
```

Comment:
/nit i feel like ? or | undefined is enough, is there any reason to include both other than being super explicit?

Reply:
Ah I should have cleaned this up, I will get it with my next PR.  Personally I prefer the ? on the param name than undefined.  I don't have a strong preference